zulipterminal/cli/run.py

```diff
@@ -321,16 +335,19 @@ def main(options: Optional[List[str]]=None) -> None:
                            format(", ".join(complete))))
         print("   autohide setting '{}' specified {}."
               .format(*zterm['autohide']))
-        print("   footlinks setting '{}' specified {}."
-              .format(*zterm['footlinks']))
+        if zterm['footlinks'][1] == ZULIPRC_CONFIG:
+            print("   footlinks setting '{}' specified {}."
+                  .format(*zterm['footlinks']))
+        else:
+            print("   maximum footlinks setting '{}' specified {}."
+                  .format(*zterm['maximum-footlinks']))
```

Comment:
We output this as just a maximum value in the about popup; we should make them consistent.

We could output the maximum and append an element to the last part of the zterm tuple to indicate it came from the footlinks option?

Reply:
That could be done, but then it would force another if clause during the traversing of zterm, and again another to check the length of the tuple during printing. Wouldn't a bit of hard-coded values and an if-else during printing than that approach?